tests/e2e/pageobjects/openshift/OcpWebConsolePage.ts

```diff
@@ -47,104 +42,104 @@ export class OcpWebConsolePage {
         const namespaceItemInDropDownLocator: By = By.id(namespaceItemInDropDownId);
-        await this.driverHelper.waitAndClick(namespaceItemInDropDownLocator);
+        await this.driverHelper.waitAndClick(namespaceItemInDropDownLocator, TimeoutConstants.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM);
     }
 
     async clickOnSubscribeButtonOnSubscriptionPage() {
         Logger.debug('OcpWebConsolePage.clickOnSubscribeButtonOnSubscriptionPage');
 
         const subscribeOperatorButtonLocator: By = By.xpath('//button[text()=\'Subscribe\']');
-        await this.driverHelper.waitAndClick(subscribeOperatorButtonLocator);
+        await this.driverHelper.waitAndClick(subscribeOperatorButtonLocator, TimeoutConstants.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM);
     }
 
     async selectInstalledOperatorsOnNavPanel() {
         Logger.debug('OcpWebConsolePage.selectInstalledOperatorsOnNavPanel');
 
         const installedOperatorsItemNavPanelLocator: By = By.xpath('//a[text()=\'Installed Operators\']');
-        await this.driverHelper.waitAndClick(installedOperatorsItemNavPanelLocator);
+        await this.driverHelper.waitAndClick(installedOperatorsItemNavPanelLocator, TimeoutConstants.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM);
     }
 
     async waitInstalledOperatorLogoName() {
         Logger.debug('OcpWebConsolePage.waitInstalledOperatorLogoName');
 
-        await this.driverHelper.waitVisibility(By.xpath(OcpWebConsolePage.OPERATOR_LOGO_NAME_XPATH), TestConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT);
+        await this.driverHelper.waitVisibility(By.xpath(OcpWebConsolePage.OPERATOR_LOGO_NAME_XPATH), TimeoutConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT);
     }
 
     async clickOnInstalledOperatorLogoName() {
         Logger.debug('OcpWebConsolePage.clickOnInstalledOperatorLogoName');
 
-        await this.driverHelper.waitAndClick(By.xpath(OcpWebConsolePage.OPERATOR_LOGO_NAME_XPATH));
+        await this.driverHelper.waitAndClick(By.xpath(OcpWebConsolePage.OPERATOR_LOGO_NAME_XPATH), TimeoutConstants.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM);
     }
 
     async waitOverviewCsvOperator() {
         Logger.debug('OcpWebConsolePage.waitOverviewCsvEclipseCheOperator');
 
-        await this.driverHelper.waitVisibility(By.xpath(OcpWebConsolePage.OPERATOR_LOGO_NAME_XPATH));
+        await this.driverHelper.waitVisibility(By.xpath(OcpWebConsolePage.OPERATOR_LOGO_NAME_XPATH), TimeoutConstants.TS_SELENIUM_CSV_OPERATOR_TIMEOUT);
     }
 
     async clickCreateNewCheClusterLink() {
         Logger.debug('OcpWebConsolePage.clickCreateNewCheClusterLink');
 
         const createNewCheLusterLinkLocator: By = By.partialLinkText('Create');
-        await this.driverHelper.waitAndClick(createNewCheLusterLinkLocator);
+        await this.driverHelper.waitAndClick(createNewCheLusterLinkLocator, TimeoutConstants.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM);
     }
 
-    async waitCreateCheClusterYaml() {
+    async waitCreateCheClusterYaml(timeout: number) {
```

Comment:
is there a reason why this method requires the timeout to be set? Maybe have it optional? (if empty - it could boil down to https://github.com/eclipse/che/pull/17506/files#diff-c4d939529680c0b7fe915783eda8a471R91 which has a default configured, right)?

Reply:
Changed to default to TS_CREATE_CHECLUSTER_YAML_TIMEOUT.